src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -124,7 +127,57 @@ object DatabricksPlugin extends AutoPlugin {
       }
     }
   }
-  
+
+  private lazy val executeCommandImpl: Def.Initialize[Task[Unit]] = Def.taskDyn {
+    val client = dbcApiClient.value
+    val language = dbcExecutionLanguage.value
+    val onClusters = dbcClusters.value
+    val allClusters = dbcFetchClusters.value
+    val commandFile = dbcCommandFile.value
+    Def.task {
+      for (clusterName <- onClusters) {
+        // Duplicating aspects of code in DatabricksHttp.foreachCluster
+        // - this is due to inability to pass values between DatabricksHttp function
+        // calls within the foreachCluster
+        val givenCluster = allClusters.find(_.name == clusterName)
+        if (givenCluster.isEmpty) {
+          throw new NoSuchElementException(s"Cluster with name: '${clusterName} not found!")
```

Comment:
Not sure:
`logger.error` + http://www.scala-sbt.org/0.12.2/docs/Detailed-Topics/Tasks.html#handling-failure